nixos/modules/services/system/swapspace.nix

```diff
@@ -0,0 +1,106 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.swapspace;
+  inherit (lib)
+    types
+    mkOption
+    mkPackageOption
+    mkEnableOption
+    ;
+  configFile = pkgs.writeText "swapspace.conf" (
+    lib.concatStringsSep "\n" (lib.mapAttrsToList (n: v: "${n}=${toString v}") cfg.settings)
```

Comment:
Is it possible to use `lib.generators.toKeyValue ` instead?

Reply:
I've tried to add each option description as a comment in the conf file. But it was [ugly](https://github.com/phanirithvij/system/commit/15f78b9c454957cd430b5199d69a86bdb47221f8#diff-726f6e4b8f610b6251f20ec7bb8cb15fabec4a34917c8a26a5582540c4249cfcR61), I will see if it can be done with `lib.generators.toKeyValue` if not I will let it be.

Reply:
done